docker-compose.debug.yml

```diff
@@ -0,0 +1,50 @@
+version: '2'
```

Comment:
Same here, not a big fan of copy and pasting this, we should probably re-use the existing one as it is only used for development / testing. Worst-case we could add an [docker-compose.override.yml](https://docs.docker.com/compose/extends/) that just adds the modifications we want, but I don't see any reason not to try to get xdebug working for the existing local development configuration.